src/htletterspacer/__main__.py

```diff
@@ -58,18 +65,25 @@ def main(args: list[str] | None = None) -> None:
     return None
 
 
-def space_ufo(args: argparse.Namespace) -> None:
-    ufo: ufoLib2.Font = args.ufo
+def space_ufo(
+    ufo: ufoLib2.Font,
+    area: int | None = None,
+    depth: int | None = None,
+    overshoot: int | None = None,
+    config_file: Path | None = None,
+    to_space: set[str] | None = None,
```

Comment:
You could init this with `frozenset()` instead of `None` so we skip testing for it.

Reply:
When not specified, I think it should be full not empty, or do you mean in the function body?